src/Controller/Simulation.java

```diff
@@ -31,6 +34,9 @@
     private boolean running;
     private boolean LightAuto;
 
+    File logFile = new File("logFile.txt");
+    PrintWriter pw = new PrintWriter(new FileWriter(logFile, true));
+
     private boolean isAway;
```

Comment:
Might wanna move this one with the others

Reply:
cool cool